ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift

```diff
@@ -28,15 +27,18 @@ import UIKit
     public func createPaywallView() -> PaywallViewController {
         let controller = PaywallViewController()
         controller.delegate = self
-        
+        controller.dismissalDelegate = self
         return controller
     }
 
     @objc
-    public func createPaywallView(offeringIdentifier: String) -> PaywallViewController {
+    public func createPaywallView(
+        offeringIdentifier: String,
+        dismissHandler: (() -> Void)? = nil
+    ) -> PaywallViewController {
         let controller = PaywallViewController(offeringIdentifier: offeringIdentifier)
         controller.delegate = self
-
+        controller.dismissalDelegate = self
```

Comment:
We need to set this delegate for the footer methods as well. Otherwise it won't be called after a purchase completes